common/src/main/java/whocraft/tardis_refined/common/items/KeyItem.java

```diff
@@ -122,27 +122,6 @@ public static boolean keychainContains(ItemStack itemStack, ResourceKey<Level> l
         return keychain.contains(levelResourceKey);
     }
 
-    @Override
-    public InteractionResult interactLivingEntity(ItemStack itemStack, Player player, LivingEntity livingEntity, InteractionHand interactionHand) {
-
```

Comment:
Why is all this removed?

Reply:
This was not entirely removed, it was moved to the ControlEntity class. The reason for this is because  the interactLivingEntity method only has LivingEntity as a parameter. When we convert the ControlEntity to an ``Entity``, this method is no longer functional. 

What we can do is restore the logic in ``KeyItem`` and call that in ``ControlEntity``

Reply:
Addressed this. Mobed the logic to KeyItem, and called it in the Monitor Control directly.

Additionally, added logic to not open the Monitor GUI if we are syncing the Key to the current Tardis so that the player can see the hotbar message more easily.